Makefile

```diff
@@ -33,8 +33,16 @@ tox.requirements:  ## install tox requirements
 develop: test.requirements  ## install test and dev requirements
 	pip3 install -q -r requirements/dev.txt
 
+
+COMMON_CONSTRAINTS_TXT=requirements/common_constraints.txt
```

Comment:
We want to rollout this change on few IDA's with owner's team testing/review.

That's why not removing the `pyjwt` pin from common constraint but to bring latest version on `pyjwt==2.1.0` we have to do these steps. 

On make upgrade It's download the common constraint locally and paste its content in local constraint file and then remove the `pyjwt` pin.

Reply:
I am both impressed by the creative solution, and worried that this might not be cleaned up when we actually remove the pyjwt constraint. Anyways. I accept this for now.